### PR TITLE
clean up formatting and styles, typos in captools

### PR DESCRIPTION
**Pull request by dave91x**
There were some typos and other formatting/style issues that had been bugging me, especially with this being the code that we are handing off to folks as an example for using our API.

### CODE REVIEW

File: captools/api/client.py [hunk 1]
```diff
@@ -198,15 +229,17 @@ def _put_or_post_json(self, method, url, data):
...
         regex_tokens = _split_regex(regex)
         result = ''
         for i in range(len(arguments)):
             result = result + str(regex_tokens[i]) + str(arguments[i])
-        if len(regex_tokens) > len(arguments): result += regex_tokens[-1]
+        if len(regex_tokens) > len(arguments):
+            result += regex_tokens[-1]
         return result
-        #return '%s://%s/%s' % (self.parsed_endpoint.scheme, self.parsed_endpoint.netloc, result)
+        # return '%s://%s/%s' % (self.parsed_endpoint.scheme, self.parsed_endpoint.netloc, result)
```
Comment (yorinasub17) [inline captools/api/client.py]: lets delete this too, since it's dead code.

File: setup.py [hunk 2]
```diff
@@ -32,24 +33,25 @@
...
-        classifiers=[
-            "Programming Language :: Python",
-            "License :: OSI Approved :: MIT License",
-            "Development Status :: 3 - Alpha",
-            "Intended Audience :: Developers",
-        ],
-        long_description = LONG_DESCRIPTION,
-        packages=['captools', 'captools.api'],
-        package_data={'captools.api': ['img/*.png']})
+      version='0.20',
```
Comment (yorinasub17) [inline setup.py]: lets bump the version too.

**Comment by yorinasub17** [comment]
LGTM except for comments. When merge when addressed, and then update PyPI. Thanks for taking care of this!

**Comment by dave91x** [comment]
@yorinasub17 
Thanks for reviewing.  I made those two changes you mentioned.  I also tried to run python tests.py in the captools/api folder and got the results shown below.  I ran it against a local dev shreddr setup in which I had created a regular user and generated an API token for it, which I put in the tests.py file.  I'm not exactly sure why it is having problems.  Have the tests in tests.py been kept up to date with the changes in Shreddr?

FE
======================================================================
ERROR: test_non_json_resource (__main__.TestClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 72, in test_non_json_resource
    datasets = client.read_datasets(completed_job['id'])
AttributeError: 'Client' object has no attribute 'read_datasets'

======================================================================
FAIL: test_client_connection (__main__.TestClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 32, in test_client_connection
    self.assertTrue(hasattr(client, 'read_' + resource1['name']))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 2 tests in 3.197s

FAILED (failures=1, errors=1)

**Comment by yorinasub17** [comment]
@dave91x Yes the tests hasn't been updated to keep up with shreddr updates, because we migrated all the tests to within Arabica `backbone.tests.test_external_api`. I just ran it with the new captools and everything was fine.

Let's remove that tests.py file to avoid confusion, then merge away!

**Comment by yorinasub17** [comment]
LGTM!

**Pull request merged**